test/controllers/concerns/ensure_installed_test.rb

```diff
@@ -61,6 +66,36 @@ def index
     assert_response :ok
   end
 
+  test "redirects to login_url (oauth path) to reinstall the app if the store's session token is no longer valid" do
+    ShopifyApp.configuration.stubs(:embedded_app?).returns(true)
```

Comment:
I think you can actually change the configuration value as part of the tests, I'm pretty sure we reset them for each test.